CHANGES.rst

```diff
@@ -177,6 +177,8 @@ astropy.io.fits
 astropy.io.misc
 ^^^^^^^^^^^^^^^
 
+- Added QDP data file handling [#11256]
```

Comment:
This should be in `io.ascii` and can be a bit more verbose, perhaps "Added support for reading and writing ASCII tables in QDP (Quick and Dandy Plotter) format."

Reply:
Done